1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,11 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for (let salary in salaries) {
```

Comment:
your code is spot on, just remember `for...in` loops though the **keys** of an object, so the key here is the name (ross, monica and rachel), i.e.
```
for (let name in salaries){
...
```